### PR TITLE
166 UI clean up css

### LINKED ISSUE #166: UI Clean-up - CSS/FXML/java files code repartition and CSS files organization
UI Clean-up on the UI code files repartition and structure.

Acceptance Criteria:
- [x] Address all warnings in FXML files
- [x] FXML files only contain the layout properties of static UI components, the fx:ID, text values, as well as inherited-default properties that are not styling.
- [x] FXML files do not contain any styling
- [x] All static styling is done in the CSS files (no duplicate styling between the FXML and CSS files)
- [x] Old CSS files are removed
- [x] The revamp CSS files are set as the new default package
- [x] All duplicated styling among the different css files for different controllers should be removed from those specific controller css files and brought under the styling css file.
- [x] Java code files should only contain the layout and styling (if it can't be done in CSS) of dynamic UI components.
- [x] CSS Files should all be organized the same way (consistency): Divide CSS files in different sections based on the type of Element being style, Add comments for each section
- [x] A font has been decided for the project, downloaded to the repo, and used on the whole project from the downloaded location

### PR DESCRIPTION
**Pull request by jeffbuonamici**
Refactored CSS files
Removed inline styling
Changed controllers to use new styling names

This resolves #166.

### CODE REVIEW

File: JavaFXUserInterface/src/main/resources/AssetInfo.fxml [hunk 1]
```diff
@@ -106,7 +158,10 @@
         </AnchorPane>
     </children>
     <stylesheets>
-        <URL value="@css/revamp/assetInfo.css" />
-        <URL value="@css/revamp/style.css" />
+        <URL value="@css/assetInfo.css"/>
```
Comment (rdsaliba) [inline JavaFXUserInterface/src/main/resources/AssetInfo.fxml:161]: IntelliJ cannot resolve the filename 'assetInfo.css'.
There's a typo, just change it to 'assetinfo.css to match the file name :)

File: JavaFXUserInterface/src/main/resources/Assets.fxml [hunk 2]
```diff
@@ -14,31 +14,42 @@
 <?import javafx.scene.text.Text?>
```
Comment (rdsaliba) [inline JavaFXUserInterface/src/main/resources/Assets.fxml:13]: Remove unused import

File: JavaFXUserInterface/src/main/resources/AssetTypeList.fxml [hunk 3]
```diff
@@ -9,12 +9,11 @@
 <?import javafx.scene.control.TextField?>
 <?import javafx.scene.layout.AnchorPane?>
 <?import javafx.scene.layout.FlowPane?>
-<?import javafx.scene.text.Font?>
 <?import javafx.scene.text.Text?>
 <?import javafx.scene.control.TableView?>
 <?import javafx.scene.control.TableColumn?>
```
Comment (rdsaliba) [inline JavaFXUserInterface/src/main/resources/AssetTypeList.fxml:15]: There is 5 unresolved fx:id in this file
* search
* assetsTypeTabPane
* thumbnailTab
* listTab
* assetTypesListPane

Any effect?

File: JavaFXUserInterface/src/main/resources/AssetInfo.fxml [hunk 4]
```diff
@@ -1,92 +1,143 @@
...
-                                                        <Text fill="#828282" layoutX="15.0" layoutY="145.0" strokeType="OUTSIDE" strokeWidth="0.0" text="Status:" textAlignment="RIGHT" wrappingWidth="90.0" />
-                                                        <Text fx:id="recommendationOutput" layoutX="129.0" layoutY="145.0" strokeType="OUTSIDE" strokeWidth="0.0" text="T" />
-                                                        <ImageView fx:id="imageView" fitHeight="150.0" fitWidth="200.0" layoutX="867.0" layoutY="13.0" pickOnBounds="true" preserveRatio="true" />
+                                                        <Text fx:id="assetTypeLabel" fill="#828282" layoutX="15.0"
+                                                              layoutY="58.0" strokeType="OUTSIDE" strokeWidth="0.0"
+                                                              text="Asset Type:" textAlignment="RIGHT"
+                                                              wrappingWidth="90.0"/>
+                                                        <Text fx:id="serialNumber" fill="#828282" layoutX="15.0"
```
Comment (rdsaliba) [inline JavaFXUserInterface/src/main/resources/AssetInfo.fxml]: multiple unresolved fx:id reference in this file as well!

**Review by rdsaliba — CHANGES_REQUESTED** [review]
@jeffbuonamici Great work man! There's still minor warnings in the FXML file to correct but that's a hugeeee improvement. Also, the CSS looks very neat. I like how you grouped things together and removed a lode of styling duplication.

File: JavaFXUserInterface/src/main/resources/AssetInfo.fxml (hunk 1 above)
Comment (jeffbuonamici) [reply JavaFXUserInterface/src/main/resources/AssetInfo.fxml:161]: image
I don't have this issue, and it's spelt the exact same way on my end

File: JavaFXUserInterface/src/main/resources/Assets.fxml (hunk 2 above)
Comment (jeffbuonamici) [reply JavaFXUserInterface/src/main/resources/Assets.fxml:13]: done

File: JavaFXUserInterface/src/main/resources/AssetTypeList.fxml (hunk 3 above)
Comment (jeffbuonamici) [reply JavaFXUserInterface/src/main/resources/AssetTypeList.fxml:15]: done

File: JavaFXUserInterface/src/main/resources/AssetInfo.fxml (hunk 4 above)
Comment (jeffbuonamici) [reply JavaFXUserInterface/src/main/resources/AssetInfo.fxml]: done

File: JavaFXUserInterface/src/main/resources/AssetInfo.fxml (hunk 1 above)
Comment (rdsaliba) [reply JavaFXUserInterface/src/main/resources/AssetInfo.fxml:161]: image
weird....

Comment (rdsaliba) [reply JavaFXUserInterface/src/main/resources/AssetInfo.fxml:161]: ok resolved now!

Comment (jeffbuonamici) [reply JavaFXUserInterface/src/main/resources/AssetInfo.fxml:161]: Fixed, was a github issue

**Review by rdsaliba — APPROVED** [review]

**Review by najimgh — APPROVED** [review]
The acceptance criteria are met. The separation of the CSS files is well done and duplicated styling has been taken care of.
Awesome Job!

**Pull request merged**